po/hu.po

```diff
@@ -47,105 +48,105 @@ msgstr ""
 
 #: src/applicationRow.ui:57
 msgid "Application"
-msgstr ""
+msgstr "Alkalmazás"
 
 #: src/applicationRow.ui:74 src/permissionEntryRow.ui:45
 #: src/permissionSwitchRow.ui:45
 msgid "Description"
-msgstr ""
+msgstr "Leírás"
 
 #: src/permissionEntryRow.ui:29 src/permissionSwitchRow.ui:29
 msgid "Pemission"
-msgstr ""
+msgstr "Engedély"
 
 #: src/view.ui:36
 msgid "No applications found."
-msgstr ""
+msgstr "Alkalmazás nem található."
 
 #: src/window.ui:41
 msgid "Reset"
-msgstr ""
+msgstr "Visszaállítás"
 
 #: src/window.ui:47
 msgid "Reset this application permisions"
-msgstr ""
+msgstr "Alkalmazás engedélyek alaphelyzetbe állítása"
 
 #: src/model.js:35
 msgid "Access network"
-msgstr ""
+msgstr "A hálózat hozzáférése"
 
 #: src/model.js:40
 msgid "Access inter-process communications"
-msgstr ""
+msgstr "A folyamatok közötti kommunikáció hozzáférése"
 
 #: src/model.js:45
 msgid "Access X11 windowing system"
-msgstr ""
+msgstr "Az X11 ablakrendszer hozzáférése"
 
 #: src/model.js:50
 msgid "Access X11 windowing system (as fallback)"
-msgstr ""
+msgstr "Az X11 ablakrendszer hozzáférése (tartalékként)"
 
 #: src/model.js:55
 msgid "Access Wayland windowing system"
-msgstr ""
+msgstr "A Wayland ablakrendszer hozzáférése"
 
 #: src/model.js:60
 msgid "Access PulseAudio sound server"
-msgstr ""
+msgstr "A PulseAudio hangkiszolgáló hozzáférése"
 
 #: src/model.js:65
 msgid "Access D-Bus system bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus rendszer busz hozzáférése (korlátlan)"
 
 #: src/model.js:70
 msgid "Access D-Bus session bus (unrestricted)"
-msgstr ""
+msgstr "A D-Bus munkamenet busz hozzáférése (korlátlan)"
 
 #: src/model.js:75
 msgid "Access Secure Shell agent"
-msgstr ""
+msgstr "A biztonsági héj ügynök hozzáférése"
```

Comment:
Should use "Secure Shell" here.

Reply:
Done. If there is anything else like this, you are welcome to change. Don't understand why there is so much back and forth.

Don't know what and how to do a rebase. Can you please do that?

Thank you